chef/cookbooks/crowbar/recipes/crowbar-upgrade.rb

```diff
@@ -27,6 +27,43 @@
 upgrade_step = node["crowbar_wall"]["crowbar_upgrade_step"] || "none"
 
 case upgrade_step
+
+when "crowbar_upgrade"
+
+  # Disable openstack services
+  # We don't know which openstack services are enabled on the node and
+  # collecting that information via the attributes provided by chef is
+  # rather complicated. So instead we fall back to a simple bash hack
+
+  bash "disable_openstack_services" do
+    code <<-EOF
+      for i in /etc/init.d/openstack-* \
+               /etc/init.d/openvswitch-switch \
+               /etc/init.d/ovs-usurp-config-* \
+               /etc/init.d/drbd;
```

Comment:
I guess this can be done in a followup, but ovs-usurp-config-* scripts are gone, openvswitch-switch is now openvswitch (but systemd service, not init script). Not sure about drbd.